trio/_file_io/__init__.py

```diff
@@ -0,0 +1,7 @@
+__all__ = []
+
+from ._file_io import *
+__all__ += _file_io.__all__
+
+from ._path import *
+__all__ += _path.__all__
```

Comment:
I tend to prefer making modules as tiny as possible, but if you want them combined, I'll do it.